software-development-roles/software-developer/senior-developer.md

```diff
@@ -0,0 +1,153 @@
+
+\*We offer a hybrid working model, allowing for a balance between remote work and time spent in your local office. [Office locations](https://example.com/map) can be found on this map.
+
+
+
+## The Role:
+
+
+**This role aligns against ‘[Senior Software Developer](https://ddat-capability-framework.service.gov.uk/role/software-developer#senior-developer)’ from the Government Digital and Data Framework**
+
+We’re recruiting for Senior Software Developers here at Justice Digital, to join one of our dynamic and collaborative teams.
+
+- **About Us:**
+  At Justice Digital, we're dedicated to leveraging technology to drive impactful change across the justice system. As a Senior Software Developer, you'll play a pivotal role in enhancing access to justice and improving outcomes for offenders through innovative digital solutions.
+
+- **Responsibilities:**
+  You’ll collaborate closely with fellow developers on our diverse tech stack and embrace opportunities to learn new technologies and systems. We tailor team assignments to align with your skills, interests, and career aspirations. For instance:
+  
+  - Reducing reoffending rates by improving the [scheduling of prison visits](https://github.com/ministryofjustice/visit-scheduler) in our Prisons team
+  - Enhancing access to justice by developing a service to [apply for legal aid](https://github.com/ministryofjustice/laa-apply-for-criminal-legal-aid) in the Legal Aid Agency
+  - Building a no-code platform for publishing government forms and services to the internet in the [MoJ Forms](https://moj-forms.service.justice.gov.uk/) team
+  - Improving outcomes for offenders by building tools to [manage rehabilitation services](https://github.com/ministryofjustice/hmpps-interventions-ui) in our Probation team
+  - Maintaining transparency of government by managing [freedom of information requests](https://github.com/ministryofjustice/correspondence_tool_public) and [parliamentary questions](https://github.com/ministryofjustice/parliamentary-questions) in our Central team
+
+- **Collaboration:**
+  You’ll collaborate closely with user researchers, product managers, designers, delivery managers, technical architects, and content specialists who share our vision of leveraging technology to transform government services.
+
+- **Discover More:**
+  To get a glimpse into life at MoJ Justice Digital, explore our [blog](https://example.com/blog) and read about our [Digital and Technology Strategy 2025](https://example.com/strategy).
+
+## Our Tech Stack:
+
+- **Technologies:**
+  We use a diverse range of technologies, and we’re seeking individuals who specialize in one or more and are eager to learn new languages and frameworks. Our tech stack includes:
+  
+  - Application Code: Ruby, Kotlin, Java, Python, NodeJS, PHP, Wordpress
+  - CI/CD: Circle CI, GitHub Actions
+  - Monitoring: Sentry
+  - Containerization: Kubernetes, Docker
+  - Cloud Infrastructure: AWS
+  - Web Interfaces: GOV.UK Design System

```

Comment:
is it worth adding a mention to MoJ Design system?

Reply:
Added 👍